src/mixins/getParticipants.js

```diff
@@ -75,7 +75,7 @@ const getParticipants = {
 		 */
 		onJoinedConversation() {
 			this.$nextTick(() => {
-				this.cancelableGetParticipants()
+				this.debounceUpdateParticipants()
```

Comment:
Should be okay to refresh on join once, so you see yourself, or is one-self hard coded already?

Reply:
User (self) is in the list already. And another fetch would be after participants list refresh (as self has joined), so don't see a reason to do it once more. But with that now we could skip one request